salt/_pillar/metalk8s_private.py

```diff
@@ -14,36 +14,39 @@ def __virtual__():
     return __virtualname__
 
 
+def _read_private_key(key_name, key_path):
```

Comment:
Could explain in the commit message why this reword (TOCTOU, obvious in the context of this review but not when finding the commit alone with a git blame)

Reply:
Done